sdk/core/tests.yml

```diff
@@ -4,3 +4,4 @@ stages:
   - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
       ServiceDirectory: core
+      BuildTargetingString: '*'
```

Comment:
```suggestion
      BuildTargetingString: 'azure*'
```
nit

Reply:
Need the '*' so `corehttp` isn't excluded.

Reply:
ah it's not `azure-corehttp`. GOT IT.